examples/README.md

```diff
@@ -60,46 +60,61 @@ Rossmann operates over 3,000 drug stores across seven European countries. Histor
+Depending on which example you want to run, you should use any one of these Docker containers:
 
-- Merlin-Tensorflow-Training (contains NVTabular with TensorFlow)
-- Merlin-Pytorch-Training (contains NVTabular with PyTorch)
-- Merlin-Training (contains NVTabular with HugeCTR)
-- Merlin-Tensorflow-Inference (contains NVTabular with TensorFlow and Triton Inference support)
+- `merlin-hugectr` (contains NVTabular with HugeCTR)
+- `merlin-tensorflow` (contains NVTabular with TensorFlow)
+- `merlin-pytorch` (contains NVTabular with PyTorch)
+
+Beginning with the 22.06 release, each container includes the software for training models and performing inference.
 
 To run the example notebooks using Docker containers, do the following:
 
 1. Pull the container by running the following command:
 
-   ```
+   ```sh
    docker run --gpus all --rm -it -p 8888:8888 -p 8797:8787 -p 8796:8786 --ipc=host <docker container> /bin/bash
    ```
 
    **NOTES**:
 
-   - If you are running `Getting Started with MovieLens` , `Advanced Ops with Outbrain` or `Tabular Problems with Rossmann` example notebooks you need to add `-v ${PWD}:/root/` to the docker script above. Here `PWD` is a local directory in your system, and this very same directory should also be mounted to the `merlin-inference`container if you would like to run the inference example. Please follow the `start and launch triton server` instructions given in the inference notebooks.
-   - If you are running `Training-with-HugeCTR` notebooks, please add `--cap-add SYS_NICE` to `docker run` command to suppress the `set_mempolicy: Operation not permitted` warnings.
+   - If you are running Getting Started with MovieLens, Advanced Ops with Outbrain, or the Tabular Problems with Rossmann example notebooks, add a `-v ${PWD}:/root/` argument to the preceding Docker command.
+   The `PWD` environment variable refers to a local directory on your computer, and you should specify this same directory and with the `-v` argument when you run a container to perform inference.
+   Follow the instructions for starting Triton Inference Server that are provided in the inference notebooks.
+   - If you are running `Training-with-HugeCTR` notebooks, please add `--cap-add SYS_NICE` to the `docker run` command to suppress the `set_mempolicy: Operation not permitted` warnings.
 
-The container will open a shell when the run command execution is completed. You will have to start JupyterLab on the Docker container. It should look similar to this:
+   The container opens a shell when the run command execution is completed.
+   Your shell prompt should look similar to the following example:
 
-```
-root@2efa5b50b909:
-```
+   ```sh
+   root@2efa5b50b909:
+   ```
 
-2. If jupyter-lab is not installed, install jupyter-lab with `pip` by running the following command:
+1. Install JupyterLab with `pip` by running the following command:
```

Comment:
I think we have jupyter lab installed on our containers now by default https://github.com/NVIDIA-Merlin/Merlin/blob/027587cab360141fac7498d461c7a26ed886fe1d/docker/dockerfile.merlin#L295 - we can probably remove this section?

Reply:
I removed it from NVT, systems, and models. I'll catch Merlin on the next update.  Thanks for pointing it out.